data/pizza.json

```diff
@@ -1239,7 +1227,7 @@
 			"ingredients": [
 				"ananas", "majs", "gurka", "färska tomater", "isbergssallad", "ost", "skinka", "valfri sås"
```

Comment:
[Rhode Islandsås instead of valfri](https://sannegarden.com/meny/johanneberg/#:~:text=89Kr-,Skinka%2C%20Ananas%2C%20Gurka%2C%20Isbergssallad%2C%20Rhode%20Islands%C3%A5s%2C%20Majs%2C%20Tomater,-Kyckling%20sallad)